src/status_im/contexts/onboarding/sync_or_recover_profile/view.cljs

```diff
@@ -0,0 +1,117 @@
+(ns status-im.contexts.onboarding.sync-or-recover-profile.view
+  (:require
+    [quo.core :as quo]
+    re-frame.db
+    [react-native.core :as rn]
+    [react-native.safe-area :as safe-area]
+    [status-im.common.resources :as resources]
+    [status-im.config :as config]
+    [status-im.contexts.onboarding.new-to-status.style :as style]
+    [utils.debounce :as debounce]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn sign-in-options
+  []
+  (let [window (rf/sub [:dimensions/window])]
+    [rn/view {:style style/options-container}
+     [quo/text
+      {:style  style/title
+       :size   :heading-1
+       :weight :semi-bold}
+      (i18n/label :t/sync-or-recover-profile)]
+     [quo/small-option-card
+      {:variant      :main
+       :title        (i18n/label :t/sign-in-by-syncing)
+       :subtitle     (i18n/label :t/if-you-have-status-on-another-device)
+       :button-label (i18n/label :t/scan-sync-code)
+       :image        (resources/get-image :generate-keys)
+       :max-height   (- (:height window)
+                        (* 2 56) ;; two other list items
+                        (* 2 16) ;; spacing between items
+                        220)     ;; extra spacing (top bar)
+       :on-press     #(debounce/throttle-and-dispatch
+                       [:onboarding/navigate-to-sign-in-by-syncing]
+                       1000)}]
+     [rn/view {:style style/subtitle-container}
+      [quo/text
+       {:style  style/subtitle
+        :size   :paragraph-2
+        :weight :medium}
+       (i18n/label :t/dont-have-statatus-on-another-device)]]
+     [rn/view
+      [quo/small-option-card
+       {:variant  :icon
+        :title    (i18n/label :t/use-recovery-phrase)
+        :subtitle (i18n/label :t/use-recovery-phrase-subtitle)
+        :image    (resources/get-image :ethereum-address)
+        :on-press #(rf/dispatch [:navigate-to-within-stack
```

Comment:
It it doesn't close over values, then it's a good idea to put in a separate private var.